packages/cozy-viewer/src/Footer/FooterContent.jsx

```diff
@@ -9,18 +9,11 @@ import { makeStyles } from 'cozy-ui/transpiled/react/styles'
 
 import BottomSheetContent from './BottomSheetContent'
 import { extractChildrenCompByName } from './helpers'
-import PrintButton from '../components/PrintButton'
 
 const FooterButtons = ({
-  file,
   FooterActionButtonsWithFile = { FooterActionButtonsWithFile }
 }) => {
-  return (
-    <>
-      {FooterActionButtonsWithFile}
-      <PrintButton file={file} variant="button" />
-    </>
-  )
+  return <>{FooterActionButtonsWithFile}</>
```

Comment:
This component still has a purpose ?

Reply:
Non en effet j'ai refactorisé dans le dernier commit 👍